edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup Condition="'$(OS)|$(DotNet_Runtime)' == 'Unix|netcoreapp3.0'">
+  <PropertyGroup>
     <CompileBlocked>true</CompileBlocked>
   </PropertyGroup>
+
+  <Import Project="..\..\..\..\..\dotnetVersion.props" />
```

Comment:
edgehub binding should target netstandard because it gets pushed as public nuget package

Reply:
I made the change here. 

Elsewhere, we do the same thing for the [samples](https://github.com/Azure/iotedge/blob/master/edge-modules/functions/samples/EdgeHubTrigger-Csharp/EdgeHubTriggerCSharp.csproj#L3) in master so I also tried to do the same. I got a build warning though
```
/home/als5ev/.nuget/packages/microsoft.azure.webjobs.script.extensionsmetadatagenerator/1.1.8/build/Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator.targets(37,5): warning :     Could not evaluate 'Microsoft.CSharp.dll' for extension metadata. Exception message: Could not load file or assembly 'Microsoft.CSharp, Version=4.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. [/home/als5ev/Desktop/debug/iotedge/edge-modules/functions/samples/EdgeHubTrigger-Csharp/EdgeHubTriggerCSharp.csproj]
```

This [link](https://stackoverflow.com/questions/57794351/vb-net-azure-functions-what-does-the-microsoft-csharp-assembly-do) says it is harmless, but if we target netcoreapp3.1 the warning goes away. I think it is better to use netcoreapp3.1 to be extra safe.

Reply:
I discovered a problem using netstandard here. Building and publishing this with an output folder throws an error.

```
/usr/share/dotnet/sdk/3.1.201/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(234,5): error NETSDK1005: Assets file '/home/als5ev/Desktop/debug/iotedge/edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/obj/project.assets.json' doesn't have a target for '.NETCoreApp,Version=v3.1'. Ensure that restore has run and that you have included 'netcoreapp3.1' in the TargetFrameworks for your project. [/home/als5ev/Desktop/debug/iotedge/edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj]
```

I have tried cleaning and recloning. Any idea why this is happening?

Reply:
Fixed. The issue was that we weren't publishing libraries as netstandard2.0.